bcrhp/src/bcrhp/constants.ts

```diff
@@ -0,0 +1 @@
+export const datePickerFormat = 'yy-mm-dd';
```

Comment:
Can we make constants screaming snake case? Let's drop the "Picker" part too as we're likely to use in other places than the date picker (eg DATE_FORMAT)